src/applications/disability-benefits/disability-rating-calculator/tests/00-all-fields.cypress.spec.js

```diff
@@ -0,0 +1,116 @@
+const Timeouts = require('platform/testing/e2e/timeouts.js');
+const inputs = require('./utils/inputs.json');
+
+const componentSelector =
+  'div[data-widget-type="disability-rating-calculator"]';
+const deleteRowBtnSelector = 'button[data-e2e=delete]';
+const drcPagePath = '/disability/about-disability-ratings/';
+const inputSelector = '.ratingInput';
+const calculateBtnSelector = 'button[data-e2e=calculate]';
+const combinedRatingSelector = 'div[data-e2e="combined-rating"]';
+const clearBtnSelector = 'button[data-e2e=clearall]';
+const rowDeletionTestInput = {
+  ratings: [10, 20, 30],
+  combinedRating: {
+    exact: 50,
+    rounded: 50,
+  },
+};
+Cypress.Commands.add('fillRatings', input => {
+  const { ratings } = input;
+  const ratingsLength = ratings.length;
+
+  for (let x = 0; x < ratingsLength; x++) {
```

Comment:
Really minor point: can we use something other than `x` for the variable name here?

Reply:
Replaced this with `ratingsIndex` to be more descriptive of it's purpose.